lib/rules/index.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
   'no-action-modifiers': require('./lint-no-action-modifiers'),
   'no-attrs-in-components': require('./lint-no-attrs-in-components'),
   'no-bare-strings': require('./lint-no-bare-strings'),
+  'no-classic-invocation-component': require('./lint-no-classic-invocation-component'),
```

Comment:
I'm not 100% sure RE: using "classic" here. I absolutely agree that this is in "bike shed" territory, but I think I'd personally prefer `no-curly-component-invocation`.

Reply:
I wasn't crazy about the "classic" part either but my initial thinking was that  using it was consistent with the [syntax conversion guide](https://guides.emberjs.com/release/reference/syntax-conversion-guide/#toc_angle-bracket-syntax) which also refers to it as "classic invocation".

"curly" is more descriptive though so if you want me to rename I can.

Reply:
Yeah, I'm a bit torn here. "classic components" very specifically means `Ember.Component`, to me it would be pretty confusing to have "classic invocation" to use a different definition of "classic". For example, you can absolutely invoke `@glimmer/component` based components with curlies but they are _for sure_ not "classic components".

Lets do the swap to `no-curly-component-invocation`, and I'll open up an issue over on the guides repo to chat through the issue.

Reply:
https://github.com/ember-learn/guides-source/issues/929